### PR TITLE
git-rename-branch: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]
Cool!  But I don't even see why this tool is that useful, because `git branch -m main`, for example, has always worked perfectly for me to change the current branch's name. 🤔

**Comment by CleanMachine1** [comment]
I think it just auto pushes it to upstream, does your command do that?

**Comment by bl-ue** [comment]
Nope.  It just changes it locally.  I can see now why that would be useful, because configuring local vs. remote branches can be difficult! 😝

**Comment by CleanMachine1** [comment]
it actually doesn't set upstream.. and it doesn't push 

``` 
 git push
fatal: The current branch testupdated has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin testupdated

```

File: pages/common/git-rename-branch.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+> Rename a Git branch.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-rename-branch>.
+
+- Rename the branch you are currently on:
+
+`git rename-branch {{new_branch_name}}`
+
+- Rename a specified branch:
```
Comment (marchersimon) [inline pages/common/git-rename-branch.md:11]:
```suggestion
- Rename a specific branch:
```
To keep the pages consistent.

**Comment by CleanMachine1** [comment]
Nevermind, it does change the name and push it, but only if the branch you are changing is already on origin

File: pages/common/git-rename-branch.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/git-rename-branch.md:11]: @bl-ue what do you think?

**Review by marchersimon — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by CleanMachine1** [comment]
And with this, I would like to apply for collaborator, where do I ask?

**Pull request merged**